tests/monotouch-test/AVFoundation/AVAudioFormatTest.cs

```diff
@@ -30,8 +28,11 @@ public void TestEqualOperatorNull ()
 				Assert.IsFalse (format == null, "format == null");
 				Assert.IsFalse (null == format, "null == format");
 			}
-			Assert.IsTrue (nullFormat == null, "nullFormat == null");
-			Assert.IsTrue (null == nullFormat, "null == nullFormat");
+			using (AVAudioFormat nullFormat = null)
+			{
+				Assert.IsTrue (nullFormat == null, "nullFormat == null");
+				Assert.IsTrue (null == nullFormat, "null == nullFormat");
+			}
```

Comment:
That was not really needed, but it's not harmful (keep it).

Adding the `using` is the same as 
```csharp
AVAudioFormat nullFormat;
try {
    nullFormat = null;
    ...
} finally {
   nullFormat?.Dispose ();
}
```
and we know that `Dispose` won't be called, so just more core is generated in case an exception occurs.

Not that it's possible recent C# compiler versions (or the JIT/AOT compiler) simplify this even since it's obviously always `null`.